### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by jaacdestructo** [primetime43/Ratings-To-Plex-Ratings#8]
Log:

2024-09-06 14:35:11 - Starting update_ratings with parameters - Filepath: C:/Users/Administrator/Documents/1cd70d31-57e0-4344-9f3c-35ab04af44ea.csv, Selected Library: Movies, Values: -IMDB-: True, -LETTERBOXD-: False, -MOVIE-: True, -TVSERIES-: False, -TVMINISERIES-: False, -TVMOVIE-: False
2024-09-06 14:35:29 - Success: Found 0 media items in the CSV file. 0 Plex ratings updated.

**Comment by primetime43** [comment]
The issue for this bug was the column names in the csv file changes, causing the code below not able to find them.

``` py
def _get_selected_media_types(self, values):
        selected_media_types = []
        if values['-MOVIE-']:
            selected_media_types.append('movie')
        if values['-TVSERIES-']:
            selected_media_types.append('tvSeries')
        if values['-TVMINISERIES-']:
            selected_media_types.append('tvMiniSeries')
        if values['-TVMOVIE-']:
            selected_media_types.append('tvMovie')
        return selected_media_types
```

Fix:

``` py
def _get_selected_media_types(self, values):
        selected_media_types = []
        
        # Add media types based on selected checkboxes
        if values['-MOVIE-']:
            selected_media_types.append('Movie')
        if values['-TVSERIES-']:
            selected_media_types.append('TV Series')
        if values['-TVMINISERIES-']:
            selected_media_types.append('TV Mini Series')
        if values['-TVMOVIE-']:
            selected_media_types.append('TV Movie')
        if values.get('-SHORT-', False):
            selected_media_types.append('Short')
        if values.get('-TVEPISODE-', False):
            selected_media_types.append('TV Episode')
        
        logger.debug("Selected media types: %s", selected_media_types)
        return selected_media_types
```

**Comment by primetime43** [comment]
Fixed with release v2.1.0
